doc/index.rst

```diff
@@ -113,14 +113,17 @@ Features
 Getting started
 ---------------
 
-For getting started with PennyLane, check out some of the
-`key concepts <https://pennylane.ai/qml/concepts.html>`_ behind quantum machine
-learning, before moving on to some `introductory tutorials <https://pennylane.ai/qml/beginner.html>`_.
+For an introduction to quantum machine learning, we have several guides and resources available
+on our `QML website <https://pennylane.ai/qml/>`_, including
+`What is QML? <https://pennylane.ai/qml/whatisqml.html>`_,
+`frequently asked questions <https://pennylane.ai/faq.html>`_,
+a `glossary of key concepts <https://pennylane.ai/qml/glossary.html>`_, and a curated selection
+of `QML videos <https://pennylane.ai/qml/videos.html>`_.
```

Comment:
Note: didn't duplicate the list formatting here as in the readme, as Sphinx places too much space around each bullet list.